src/PactNet.Native/PactVerifier.cs

```diff
@@ -48,20 +48,31 @@ internal PactVerifier(IVerifierProvider verifier, PactVerifierConfig config)
         /// <returns>Fluent builder</returns>
         public IPactVerifier ServiceProvider(string providerName, Uri pactUri)
         {
-            this.verifierArgs.Add("--provider-name");
-            this.verifierArgs.Add(providerName);
-            this.verifierArgs.Add("--hostname");
-            this.verifierArgs.Add(pactUri.Host);
-            this.verifierArgs.Add("--port");
-            this.verifierArgs.Add(pactUri.Port.ToString());
+            verifierArgs.Add("--provider-name");
```

Comment:
I think we should put all the `this.` back on to remove noise in the diff. I almost didn't spot the new `SetBasePath` call below because of those.